cli/cli/commands/files/download/download.go

```diff
@@ -53,10 +52,17 @@ const (
 var FilesUploadCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisCommand{
 	CommandStr:                command_str_consts.FilesDownloadCmdStr,
 	ShortDescription:          "Download a files artifact from an enclave",
-	LongDescription:           "Download a files artifact using an identifier(name, uuid, shortened uuid) from an enclave to the host machine",
+	LongDescription:           "Download the given files artifact from the given enclave to your machine. The files artifact and enclave are specified by identifier (name, UUID, or shortened UUID). Read more about identifiers here: https://docs.kurtosis.com/reference/resource-identifier",
 	KurtosisBackendContextKey: kurtosisBackendCtxKey,
 	EngineClientContextKey:    engineClientCtxKey,
 	Flags: []*flags.FlagConfig{
+		{
+			Key: enclaveIdentifierFlagKey,
+			Usage: "The enclave identifier of the enclave in which the script or package will be ran. " +
+				"An enclave with this name will be created if it doesn't exist.",
+			Type:    flags.FlagType_String,
+			Default: autogenerateEnclaveIdentifierKeyword,
```

Comment:
I'm not sure I fully understand the behaviour here... Like, what happens if I don't specify any enclave at all?
Usually, we use the --enclave flag when the enclave name is optional. Like for `kurtosis run`, you can specify an enclave name, if you don't we autogenerate one. But here it feels the enclave is part of the set of identifier that is required to uniquely identify the file to be downloaded. Am I missing something?

Seems like what we need is the concept of mandatory CLI flag, or something like this, dunno

Reply:
This also makes sense; I just copied this from `run` and then tested the happy path. My bad! Fixing. Thanks @gbouv